love/src/components/WeatherStation/WeatherStation.module.css

```diff
@@ -94,4 +105,12 @@
 
 .windPlotSection .sectionTitle {
   grid-column: 1 / span 2;
+}
+
+.timeSeriesControls {
```

Comment:
I suggest changing line 68 from `grid-template-rows: 1fr 1fr 1fr 1fr;` to `grid-template-columns: 1fr 1fr 1fr 1fr;`